Functions/Coverage.ps1

```diff
@@ -121,9 +130,13 @@ function Resolve-CoverageInfo
 
     $path = $UnresolvedCoverageInfo.Path
 
+    $testsPattern = '\.tests\.ps1$'
+    $includeTests = $UnresolvedCoverageInfo.IncludeTests
+
     try
     {
-        $resolvedPaths = & $SafeCommands['Resolve-Path'] -Path $path -ErrorAction Stop
+        $resolvedPaths =  & $SafeCommands['Resolve-Path'] -Path $path -ErrorAction Stop |
+                        & $SafeCommands['Where-Object'] { if($_.Path -match $testsPattern) { $includeTests } else { $true } }
```

Comment:
This would imho be easier to understand as `$includeTests -or $_.Path -notmatch $testsPattern`. It should also be faster when tests are included because then we don't match the paths.